Graph_creation.py

```diff
@@ -0,0 +1,35 @@
+import json
+from Database.FindProject import find_project_root
+
+
+table = [[0]*937 for _ in range(937)]
```

Comment:
Powinno być to raczej dynamiczne w zależności od ilości przystanków w bazie, bo każde dodanie czegoś do bazy, w przypadku statycznego rozmiaru tabeli, wymusi ingerencję w plik database. w garbage code ujdzie, w jakościowym powinno się to zmienić, bo weź tu później wytłumacz jakiemuś monke juniorowi dlaczego jak dodał do bazy nową trasę to mu się wywala xd.